requirements.in

```diff
@@ -1,26 +1,22 @@
-# normally pulled in by kombu, but version 2.2.0 is broken
-amqp<=2.1.4
-# also handles Celery as a requirement
-selinon[celery]
-sqlalchemy
-psycopg2
-lxml
-beautifulsoup4
-# We install a patched version from dnf as we cannot use requests from PyPI - we need own certificates
-#requests
-anymarkup
-jsl
-jsonschema
-unidiff
 requests
-requests-futures
+anymarkup
+beautifulsoup4
+boto3
+botocore
+flexmock
 git2json
 gitpython
-# Amazon AWS SQS
-# Celery transparently uses boto
-boto
-boto3
-semantic_version
-radon==3.0.1
-watchdog
+jsl
+jsonschema
+lxml
+pyyaml
 raven
+radon<=3.0.1
```

Comment:
Dev requirements should go to test/requiremnents.{in,txt}. Can you follow the check list created for this work? CC: @dgpatelgit